Makefile

```diff
@@ -13,4 +13,7 @@ lint:
 	# lint examples
 	flake8 --toml-config $(word 1, $(wildcard $(SERVICES_DIR)/*))/pyproject.toml --black-config $(word 1, $(wildcard $(SERVICES_DIR)/*))/pyproject.toml examples; 
 	# lint services
-	@for f in $(shell ls ${SERVICES_DIR}); do set -e; cd ${SERVICES_DIR}/$${f};flake8 .; cd ../..; done
\ No newline at end of file
+	@for f in $(shell ls ${SERVICES_DIR}); do set -e; cd ${SERVICES_DIR}/$${f};flake8 .; cd ../..; done
+
+lock:
+	@for f in $(shell ls ${SERVICES_DIR}); do set -e; cd ${SERVICES_DIR}/$${f};poetry lock --no-update; cd ../..; done
```

Comment:
When will this be used? And what is the goal?

Reply:
It is used to update the dependencies in the lockfile. It needs to be used whenever the dependencies change, e.g. a new version of the `stackit_core`

Reply:
So probably something to be triggered automatically by Renovate or something, right?

Reply:
Yes